dbms/src/TestUtils/FunctionTestUtils.cpp

```diff
@@ -115,60 +119,234 @@ void blockEqual(
     }
 }
 
+void columnToTiPBExpr(tipb::Expr * expr, const ColumnWithTypeAndName column, size_t index)
```

Comment:
how about moving `columnToTiPBExpr` and `columnsToTiPBExpr` to separate file

Reply:
Done